src/MudBlazor/Styles/components/_inputcontrol.scss

```
@@ -80,10 +80,10 @@
 
   & > .mud-input-control-input-container > .mud-input-label-inputcontrol {
     color: var(--mud-palette-text-secondary);
-    padding: 0;
+    padding: 0px;
```

Tiny nitpick, should be fully reverted

```suggestion
    padding: 0;
```

---

Off course ^^ I removed px again